vertical-pod-autoscaler/e2e/v1/admission_controller.go

```diff
@@ -532,6 +536,14 @@ func startDeploymentPods(f *framework.Framework, deployment *appsv1.Deployment)
 
 	err = framework_deployment.WaitForDeploymentComplete(c, deployment)
 	gomega.Expect(err).NotTo(gomega.HaveOccurred(), "when waiting for empty deployment to create")
+	// We want to ensure that test is not flaky due to the etcd lag.
```

Comment:
I think a short explanation of the problem would be in place. Something like:

"If admission controller  recieves pod before controller it will not apply recommendation and test will fail. Wait after creating deployment to ensure VPA knows about it, then scale up. This is not a problem in terms of correctness:..."

Reply:
Done.
I've also updated const description. It's about watch on apiserver, not a watch on etcd.